feature-detects/websockets.js

```diff
@@ -25,5 +25,11 @@
 }
 !*/
 define(['Modernizr'], function(Modernizr) {
-  Modernizr.addTest('websockets', 'WebSocket' in window && window.WebSocket.CLOSING === 2);
+  var supports;
+  try {
+    supports = 'WebSocket' in window && window.WebSocket.CLOSING === 2;
+  } catch (e) {
+    supports = false;
```

Comment:
You don't need to rest it here. Just set the initial variable value to false, and do an empty catch. Can you update and squash?